playbooks/csi_driver_install.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- name: CSI Driver installation
+  hosts: bastion[0]
+  tasks:
+    - include_role:
+        name: csi-driver-install
+      vars:
+        IBMCLOUD_API_KEY: IBMCLOUD_API_KEY: "{{ lookup('env', 'IBMCLOUD_API_KEY') }}"
+      when: (lookup('env', 'IBMCLOUD_API_KEY') != "")
```

Comment:
This way the playbook will run without error even when user forgets to export the key.

You should pass the API key as given and put a debug task in the role to check and fail when `IBMCLOUD_API_KEY` is not given.